### PR TITLE
App components

### PR DESCRIPTION
**Pull request by mwenyoa**
### The following  features were implemented in the system:
- Deleted all the boilerplates from CRA (text, images, styles).
- Created a directory called components.
- Inside components, I created a new Calculator.js file.
- I developed just UI of the calculator; 
- Used class-based components.
- Once the Calculator was ready, imported it and used it in the main component (App.js).
- Finally, I tested it in the browser;  and it rendered without problems.
> - In Calculator.js, I created a React component that matches this design:

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (Gambit142) [inline src/App.js]: - [ ] The `App` component should be a class-based component and not a functional component. Kindly make this change. 👍

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    return (
+      <div className="calculator-container d-flex flex-d-c">
+        <div className="calculator-output d-flex j-content-flex-end">0</div>
+        <div className="d-flex flex-d-c">
+          <div className="d-flex">
+            <button type="button" className="operator operatorTop">AC</button>
+            <button type="button" className="operator operatorTop">+/-</button>
+            <button type="button" className="operator operatorTop">%</button>
+            <button type="button" className="operator operatorTop operatorRight">*</button>
+          </div>
```
Comment (Gambit142) [inline src/components/Calculator.js]: - [ ] Kindly make this the divide symbol `÷` and not the multiplication symbol `*`. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @mwenyoa 👋, 

Good job so far!

## Highlights
- Created a `components` directory 💯 
- Created `Calculator` component 💯 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly remove the following files that were created when you ran the `create react app` command:
  - public/manifests.json
  - public/robots.txt
  - src/App.test.js
  - src/setupTests.js

- _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Calculator.js (hunk 2 above)
Comment (mwenyoa) [reply src/components/Calculator.js]: @Gambit142  Thank you for the good and clear feedback. Let me work on them and submit them for a re-review.

Comment (Gambit142) [reply src/components/Calculator.js]: @mwenyoa, I am glad I could be of service. 🙇

**Review by gemmen29 — APPROVED** [review]
Hi @mwenyoa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] [OPTIONAL] Kindly fix the size of the division symbol because now it's bigger than other symbols.
image 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**